src/main/java/build/buildfarm/cas/cfc/CASAccessMetricsRecorder.java

```diff
@@ -0,0 +1,242 @@
+package build.buildfarm.cas.cfc;
```

Comment:
This does not belong in the cas (let alone cfc) package - a more appropriate location is a peer to the metrics presentation endpoint. Moving it out will alleviate the backplane interface bleed noted above.

Reply:
Moved under metrics package.